index.html

```diff
@@ -63,6 +60,171 @@ <h2>I'am a Software Developer</h2>
+                <li>Ruby</li>
+            </ul>
+    
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
+        <div class="project project3">
+            <div class="content">
+                <h2 class="contenthead">Professional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="projectlinks">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
+        <div class="project project4">
+            <div class="content">
+                <h2 class="contenthead">Professional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="projectlinks">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+       
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
+
+        <div class="project project5">
+            <div class="content">
+                <h2 class="contenthead">Professional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="projectlinks">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+    
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
+
+        <div class="project project6">
+            <div class="content">
+                <h2 class="contenthead">Professional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="projectlinks">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
+        <div class="project project7">
+            <div class="content">
+                <h2 class="contenthead">Professional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="projectlinks">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
```

Comment:
Kindly replace the `main` tag with a `section`. This block of code represents a generic standalone section of the document so it's semantically correct to use a `section` tag. :nerd_face: